res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -766,13 +770,16 @@ NumarkMixtrack3.deck.prototype.focusNextEffect = function() {
 
 NumarkMixtrack3.sampler = function(decknum) {
     this.decknum = decknum;
-    this.group = "[Sampler" + decknum + "]";
+    this.group = '[Sampler' + decknum + ']';
     this.loaded = false;
     this.PitchFaderHigh = 0;
     this.lastfadervalue = 0;
     this.LEDs = [];
 
     this.PADSampleButtonHold = new LongShortBtn(NumarkMixtrack3.onPADSampleButtonHold);
+
+    engine.softTakeover(this.group, 'pregain', true);
```

Comment:
This is manipulated by a relative control (Beats encoder), so soft takeover doesn't matter.